plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -87,10 +84,10 @@ const getNextWorkflows = (
 
 const useStyles = makeStyles(_ => ({
   topRowCard: {
-    height: '258px',
```

Comment:
change this to rem

Reply:
It turns out that the `rem`s are set to be a scale of 10 pixels, so changing this to `2.58rem` will yield the same result.

Reply:
actually, rems are relative to the default font size and pixels are absolute
the default font size is 16px so 1 rem is by default 16px
However, if you change the default font size, using pixels, you won't get the scaling relative to the font size. 
So this is an accessitiliby issue.
For example, I changed my font size in chrome settings to be very large.
This is the details card I see:
![image](https://github.com/janus-idp/backstage-plugins/assets/22211154/d750153d-6fd6-4093-8bf3-ea8d01ded76f)